VTUL/compel
# switch to sopa image and text

**Link to school of performing arts not visual arts.**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1459) (:star:)

# What does this Pull Request do? (:star:)
Change image, text and hyperlink in footer to VT's School of Performing Arts

# What's the changes? (:star:)
Change image, text and hyperlink in footer to VT's School of Performing Arts

# How should this be tested?

* Visit any page on the COMPEL site and confirm that the footer has been changed to School of Performing Arts on the left side with their triangle-ish logo
* Confirm that the image and text links to: https://www.performingarts.vt.edu/

# Additional Notes:
* Branch: performing_arts

(:star:) Required fields
